extensions/amp-autocomplete/0.1/test/test-amp-autocomplete-bindings.js

```diff
@@ -0,0 +1,244 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import '../amp-autocomplete';
+import {AutocompleteBindingInline} from '../autocomplete-binding-inline';
+import {AutocompleteBindingSingle} from '../autocomplete-binding-single';
+import {createElementWithAttributes} from '../../../../src/dom';
+import {toggleExperiment} from '../../../../src/experiments';
+
+describes.realWin(
+  'amp-autocomplete bindings',
+  {
+    amp: {
+      extensions: ['amp-autocomplete'],
+    },
+  },
+  env => {
+    let win, doc, input;
+
+    beforeEach(() => {
+      win = env.win;
+      doc = win.document;
+      input = doc.createElement('input');
+    });
+
+    function getBindingStub(attributes) {
+      const element = createElementWithAttributes(
+        doc,
+        'div',
+        Object.assign(attributes, {layout: 'container'})
+      );
+      element.appendChild(input);
+
+      const form = doc.createElement('form');
+      form.appendChild(element);
+      doc.body.appendChild(form);
+
+      return element.hasAttribute('inline')
```

Comment:
Since you're already using a separate function to instantiate an inline binding, don't create a conditional. Rather extract a function to create the holding element and then just use that from separate binding constructors. (Also, I'm not sure the binding is a `stub`).

```js

function stubAmpAutocomplete(attributes) {
  const element = createElementWithAttributes(
    doc,
    'div',
    Object.assign(attributes, {layout: 'container'})
  );
  element.appendChild(input);
  return {element, win: env.win};
}

const getBindingSingle = attributes => 
  new AutocompleteBindingSingle(stubAmpAutocomplete(attributes));

const getBindingInline = attributes => 
  new AutocompleteBindingInline(stubAmpAutocomplete(attributes));
```